lib/CPAN/Meta/Spec.pm

```diff
@@ -362,7 +362,7 @@ Example:
 
   'meta-spec' => {
     version => '2',
-    url     => 'http://search.cpan.org/perldoc?CPAN::Meta::Spec',
```

Comment:
Will this cause breakage in merging? Should this be handled specifically in `CPAN::Meta::Merge`?